docs/topics/tour/kotlin-tour-intermediate-libraries-and-apis.md

```diff
@@ -0,0 +1,284 @@
+[//]: # (title: Intermediate: Libraries and APIs)
+
+To get the most out of Kotlin, use existing libraries and APIs so you can spend more time coding and less time 
+reinventing the wheel.
+
+Libraries distribute reusable code that simplifies common tasks. Within libraries, there are packages and objects that
+group related classes, functions, and utilities. Libraries expose APIs (Application Programming Interfaces) as a set of 
+functions, classes, or properties that developers can use in their code.
+
+Let's explore what's possible with Kotlin.
+
+## The standard library
+
+Kotlin has a standard library that provides essential types, functions, collections, and utilities to make your code 
+concise and expressive. A large portion of the standard library (everything in the [`kotlin` package](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin/)) is readily available
+in any Kotlin file without you having to explicitly import it:
+
+```kotlin
+fun main() {
+    val text = "emosewa si niltoK"
+    
+   // Use the reversed() function from the standard library
+    val reversedText = text.reversed()
+
+    // Use the print() function from the standard library
+    print(reversedText)
+    // Kotlin is awesome
+}
+```
+{kotlin-runnable="true" id="kotlin-tour-libraries-stdlib"}
+
+But there are some parts of the standard library that require you to import them before using them in your code. 
+For example, if you want to use the standard library's time measurement features, you need to import the [`kotlin.time` package](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.time/).
+
+At the top of your file, or before where you use it in your code, add the `import` keyword followed by the package that you need:
```

Comment:
Imports are always at the top of the file or I didn't understand the meaning

or maybe you meant "Before using it in your code, at the top of the file, add the `import` keyword followed by the package that you need:"

Reply:
You're right. Fixed!